### PR TITLE
[60-e] [studio-ui] block unauthorized roles from accessing the Site Config

### PR DESCRIPTION
**Pull request by dejan-brkic**
fixed issue with accessing site config consule using direct url

### CODE REVIEW

**Review by sumerjabri — APPROVED** [review]

**Review by russdanner — COMMENTED** [review]
This code leaves the user looking at a blank screen. Why not redirect them as shown in the example? @dejan-brkic

File: templates/web/site-config.ftl [hunk 1]
```diff
@@ -16,6 +16,11 @@
 
 <!doctype html>
 <html xmlns="http://www.w3.org/1999/xhtml">
+
+<#if !(envConfig.role! == 'admin' || envConfig.role! == 'developer')>
+  <script>window.location='/studio/site-dashboard'</script>
+</#if>
```
Comment (rart) [inline templates/web/site-config.ftl:22]: I don't think client-side redirection is sufficient. Still allows viewing the source of the page (e.g. in chrome, you can do `view-source:localhost:8080/studio/site-config`). People could also disable JS, visit the page and reenable and start console manipulations. We should redirect server-side (e.g. freemarker util, groovy controller, headers). @sumerjabri @dejan-brkic

Comment (rart) [reply templates/web/site-config.ftl:22]: In any case, this should be first line of the head.

Comment (rart) [reply templates/web/site-config.ftl:22]: Here's an enhancement https://github.com/craftercms/studio-ui/pull/1636; Nevertheless, if we had a server-side mechanism we can use now, we should opt for that.

**Pull request merged**